geode-core/src/main/java/org/apache/geode/StatisticsTypeFactory.java

```diff
@@ -156,6 +161,9 @@ StatisticDescriptor createDoubleCounter(String name, String description, String
   /**
    * Creates and returns an int gauge {@link StatisticDescriptor} with the given <code>name</code>,
    * <code>description</code>, <code>largerBetter</code>, and <code>units</code>.
+   *
+   * @deprecated as of Geode 1.10, use {@link #createLongGauge(String, String, String, boolean)}
+   *             instead
    */
   StatisticDescriptor createIntGauge(String name, String description, String units,
```

Comment:
Can you use the method annotation `@Deprecated` too.

Reply:
done